backend/src/packages/trucks/truck.service.ts

```diff
@@ -66,6 +68,33 @@ class TruckService implements IService {
 
     return result.map((element) => TruckEntity.initialize(element).toObject());
   }
+
+  public async addTrucksToUser(
+    userId: number,
+    trucksId: number[],
+  ): Promise<{ userId: number; trucksId: number[] }> {
+    const userTrucks: number[] = [];
+
+    for (const truckId of trucksId) {
+      const [result] = await this.repository.addUser(userId, truckId);
+      userTrucks.push(result.truckId);
+    }
```

Comment:
the number of db calls insertion will depend on how much trucks driver can have access to - bad approach
better to construct array of values to insert, and insert multiple rows only during 1 db call

also userTrucks array will be the array of inserted truckIds, so same array as coming one in parameters. ( same ad trucksIds: number[]) - don't need it
![image](https://github.com/BinaryStudioAcademy/bsa-2023-towhub/assets/61375322/d489453b-5e54-4521-af93-5195e4dd629a)